src/prototypes/draft-email-textarea/demo/demo.scss

```diff
@@ -0,0 +1,142 @@
+@use '../../../compiled/tokens/scss/ease';
+@use '../../../compiled/tokens/scss/transition';
+@use '../../../compiled/tokens/scss/z-index';
+@use 'sass:color' as sass-color;
+@use '../../../mixins/ms';
+
+///
+/// Tooltip styles
+///
+/// These Tooltip styles will be inlined in a Gutenberg Custom HTML block
+///
+/// TODO Consider making a Tooltip patterns component in the future
+///
+
+$tooltip-color-background: var(--theme-color-background-secondary);
+$tooltip-color-border: #{sass-color.change(
+    color.$brand-primary-darker,
+    $alpha: 0.4
+  )};
+
+///
+/// 1. Horizontally centers tooltip
+/// 2. Positions tooltip vertically (below button)
+///
+
+._c-tooltip {
+  background: $tooltip-color-background;
+  border: ms.step(-4) solid $tooltip-color-background;
+  border-radius: size.$border-radius-medium;
+  box-shadow: 0 0 0 size.$edge-medium $tooltip-color-border,
+    size.$edge-large size.$edge-large 0 size.$edge-medium #{sass-color.change(
+        color.$brand-primary-darker,
+        $alpha: 0.2
+      )};
+  color: var(--theme-color-text-emphasis);
+  inset-block-start: 100%; // 2
+  inset-inline-start: 50%; // 1
+  opacity: 0;
+  position: absolute; // 1
+  transform: translate(-50%, 0); // 1, 2
+  white-space: nowrap;
+  z-index: z-index.$alert;
+
+  ///
+  /// Tooltip arrows (background and foreground)
+  ///
+  /// 1. Horizontally center arrows
+  ///
+
+  &::before,
+  &::after {
+    border-style: solid;
+    content: '';
+    inset-inline-start: 50%; // 1
+    position: absolute; // 1
+    transform: translateX(-50%); // 1
+  }
+
+  ///
+  /// Background tooltip arrow (darker color)
+  ///
+  /// 1. Combined with foreground arrow, matches border-width of tooltip body
+  /// 2. Position arrow above tooltip body
+  ///
+
+  &::before {
+    border-color: transparent transparent $tooltip-color-border;
+    border-width: 13px; // 1
+    inset-block-start: -35px; // 2
+  }
+
+  ///
+  /// Background tooltip arrow (lighter color)
+  ///
+  /// 1. Combined with background arrow, matches border-width of tooltip body
+  /// 2. Position arrow above tooltip body
+  ///
+
+  &::after {
+    border-color: transparent transparent $tooltip-color-background;
+    border-width: 10px; // 1
+    inset-block-start: -28px; // 2
+  }
+
+  ///
+  /// Shared animation state properties
+  ///
+
+  &.is-animating-intro,
+  &.is-animating-outro {
+    animation-duration: transition.$slow;
+    animation-fill-mode: both;
+  }
+
+  ///
+  /// Tooltip intro animation config
+  ///
+
+  &.is-animating-intro {
+    animation-name: tooltipIntro;
+  }
+
+  ///
+  /// Tooltip outro animation config
+  ///
+
+  &.is-animating-outro {
+    animation-name: tooltipOutro;
+    animation-timing-function: ease.$out_sine;
+  }
+}
+
+///
+/// Tooltip intro animation
+///
+
+@keyframes tooltipIntro {
+  from {
+    animation-timing-function: ease.$out_sine;
+  }
+
+  50% {
+    animation-timing-function: ease.$in_out_sine;
+    transform: translate(-50%, 15%);
```

Comment:
I was confused for a second about why the -50% horizontal translation was necessary. A comment might be helpful here.

Reply:
I added another potential comment. Not sure if it's better or worse.